ext/ImplicitDifferentiationChainRulesExt.jl

```diff
@@ -4,11 +4,16 @@ using AbstractDifferentiation: ReverseRuleConfigBackend, pullback_function
 using ChainRulesCore: ChainRulesCore, NoTangent, RuleConfig, ZeroTangent, unthunk
 using ImplicitDifferentiation: ImplicitFunction, PullbackMul!, check_solution
 using LinearOperators: LinearOperator
+using SimpleUnPack: @unpack
 
 """
-    rrule(rc, implicit, x[; kwargs...])
+    rrule(rc, implicit, x; kwargs...)
+    rrule(rc, implicit, x, Val(return_byproduct); kwargs...)
```

Comment:
I think this way of showing the call signature is clearer for users who don't know what a `Val` is